### PR TITLE
some minor update

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
Signed-off-by: xiang <xiang13225080@163.com>

1. add/update JSON label for config
2. set default values for server mode

### CODE REVIEW

**Comment by cwen0** [comment]
@WangXiangUSTC Is this PR already available for review?

**Comment by WangXiangUSTC** [comment]
> @WangXiangUSTC Is this PR already available for review?

available for review now @cwen0 @Andrewmatilde PTAL

File: cmd/attack/network.go [hunk 1]
```diff
@@ -209,7 +209,7 @@ func NetworkDNSCommand(dep fx.Option, options *core.NetworkCommand) *cobra.Comma
 
 	cmd.Flags().StringVarP(&options.DNSServer, "dns-server", "", "123.123.123.123",
 		"update the DNS server in /etc/resolv.conf with this value")
-	cmd.Flags().StringVarP(&options.DNSHost, "dns-hostname", "H", "", "map this host to specified IP")
+	cmd.Flags().StringVarP(&options.DNSDomainName, "dns-domain-name", "d", "", "map this host to specified IP")
```
Comment (cwen0) [inline cmd/attack/network.go:212]: Can we need to update the document?

Comment (WangXiangUSTC) [reply cmd/attack/network.go:212]: https://chaos-mesh.org/zh/docs/next/simulate-network-chaos-in-physical-nodes/#dns-%E6%95%85%E9%9A%9C%E7%9B%B8%E5%85%B3%E9%85%8D%E7%BD%AE%E8%AF%B4%E6%98%8E
Already update, and I will check all the updated config in the document later

Comment (cwen0) [reply cmd/attack/network.go:212]: image

**Review by cwen0 — APPROVED** [review]
LGTM

File: cmd/attack/network.go (hunk 1 above)
Comment (WangXiangUSTC) [reply cmd/attack/network.go:212]: OK, I will fix it

**Review by Andrewmatilde — APPROVED** [review]
LGTM

**Pull request merged**